base/src/main/java/com/smartdevicelink/managers/screen/BaseSoftButtonManager.java

```diff
@@ -136,7 +136,11 @@ public void onCapabilityRetrieved(Object capability) {
                     for (WindowCapability windowCapability : mainDisplay.getWindowCapabilities()) {
                         int currentWindowID = windowCapability.getWindowID() != null ? windowCapability.getWindowID() : PredefinedWindows.DEFAULT_WINDOW.getValue();
                         if (currentWindowID == PredefinedWindows.DEFAULT_WINDOW.getValue()) {
-                            softButtonCapabilities = windowCapability.getSoftButtonCapabilities().get(0);
+                            if (windowCapability.getSoftButtonCapabilities() != null) {
```

Comment:
Should probably also check that the array is not empty since the next line tries to access an item directly.